index.html

```diff
@@ -2535,7 +2534,7 @@
               [[!RFC3711]]. Since symmetric RTP [[!RFC4961]] is utilized, the
               <a>DTLS-SRTP</a> session is bi-directional.</p>
               <p>If <var>remoteParameters</var> is invalid, <a>throw</a> an
-              <code>InvalidParameters</code> exception. If <code>start()</code> is called
+              <code>InvalidAccessError</code>. If <code>start()</code> is called
```

Comment:
InvalidAccessError is [deprecated](https://github.com/heycam/webidl/pull/246). We may want to file a bug to purge it from WebRTC 1.0, but maybe it's too late.